main/cache/cache.cc

```diff
@@ -9,6 +10,15 @@
 using namespace std;
 
 namespace sorbet::realmain::cache {
+
+namespace {
+unique_ptr<KeyValueStore> openCache(std::shared_ptr<::spdlog::logger> logger, std::string flavor, std::string cacheDir,
+                                    size_t maxCacheSizeBytes) {
+    return make_unique<KeyValueStore>(std::move(logger), sorbet_full_version_string, std::move(cacheDir),
+                                      std::move(flavor), maxCacheSizeBytes);
+}
```

Comment:
This isn't as compelling as when I first wrote it, I'd be happy to inline it at the call sites.